examples/morphing_basis/morphing_example.ipynb

```diff
@@ -0,0 +1,388 @@
+    "# The other functions are the is the same as the previous example\n",
+    "# We can get the morphing functions, weights, and xsec values\n",
+    "morpher_2_matrix = morpher_2.calculate_morphing_matrix()\n",
+    "\n",
+    "# We input the same theta value as the previous example\n",
+    "morpher_2_weight = morpher_2.calculate_morphing_weights(theta=[1, 1])\n",
+    "\n",
+    "# calculate the predict value of point [1,1] with six basis points\n",
+    "n_base_6 = calculate_predict_xsec(morphing_weights=morpher_2_weight, xsec=xsec_simulated)\n",
+    "\n",
+    "# We compare the results of the regular and overdetermined morphing with simulated value\n",
+    "\n",
+    "print(\"simulated value:\", xsec_simulated[6])\n",
+    "print(\"regular predict value\", n_base_5)\n",
+    "print(\"overdetermined predict value:\", n_base_6)\n",
+    "\n",
+    "# We can see with only one additional basis point, the predict value is much more close to the simulated value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Below is a table showing the results with different number of basis points(5/7) of the example above. We can see that the more points and the corresponding points are provided, the better prediction results we will get from the morphing. \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\"\n",
+    "+----+-----------+---------------------+---------------------+\n",
+    "| g2 | simulated |      n_base = 5     |      n_base = 7     |\n",
+    "+----+-----------+---------------------+---------------------+\n",
+    "| -5 |   0.759   |  0.7590000033377339 |  0.7590043565724045 |\n",
+    "| -4 |    0.53   |  0.5299999713898771 |  0.5300930421799421 |\n",
+    "| -3 |    0.4    | 0.40000000596042407 | 0.39946969877928495 |\n",
+    "| -2 |   0.335   | 0.33500000834464144 |  0.3361039091832936 |\n",
+    "| -1 |   0.316   |  0.3160000145435327 | 0.31487447768449783 |\n",
+    "| 0  |   0.316   | 0.33900004625320435 |  0.3165692798793316 |\n",
+    "| 1  |   0.328   | 0.41500043869018555 | 0.32788529992103577 |\n",
+    "| 2  |    0.34   |  0.5700044631958008 | 0.34142857789993286 |\n",
+    "| 3  |   0.354   |  0.8450050354003906 | 0.35571449995040894 |\n",
+    "| 4  |   0.364   |  1.2960090637207031 |  0.3751673698425293 |\n",
+    "| 5  |   0.376   |  1.9940032958984375 |  0.4101197123527527 |\n",
+    "| 7  |   0.4205  |   4.49005126953125  |  0.5974075496196747 |\n",
+    "| 9  |   0.5347  |  9.201080322265625  |  1.1184179782867432 |\n",
+    "| 11 |   0.7822  |  17.23541259765625  |  2.2685461044311523 |\n",
+    "| 13 |   1.244   |  29.94012451171875  |  4.4378204345703125 |\n",
+    "+----+-----------+---------------------+---------------------+\n",
+    "\"\"\"\n",
+    "print()"
```

Comment:
I would suggest converting this to a comment field with a Markdown table since you do not use any of the content directly.

Reply:
Sure, I changed it to the markdown table in the new commit